src/api/Flooq.IntegrationTest/VersionTest.cs

```diff
@@ -0,0 +1,32 @@
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+using Factory = Flooq.IntegrationTest.FlooqWebApplicationFactory<Program>;
+using Version = Flooq.Api.Models.Version;
+
+namespace Flooq.IntegrationTest;
+
+[TestClass]
+public class VersionTest
+{
+  private HttpClient _client;
+  
+  [TestInitialize]
+  public void Setup()
+  {
+    _client = Factory.Factory.CreateClient();
+  }
+  
+  [TestMethod]
+  public async Task CanGetVersion()
```

Comment:
Can you add a test case if no version exists?

Reply:
Not sure because the test version is filled into the test database on creating of the test web app, and I don't see any way to remove it afterwards.